src/Compilers/Core/MSBuildTask/InteractiveCompiler.cs

```diff
@@ -259,7 +261,7 @@ protected virtual void AddResponseFileCommands(CommandLineBuilderExtension comma
                 }
             }
 
-            if (ResponseFiles != null)
+            if (ScriptResponseFiles != null)
```

Comment:
>ScriptResponseFiles [](start = 16, length = 19)

thanks for catching this. Will probably need a test for semantic change

Reply:
Where would this test go? I can't find any tests for `ResponseFiles` or `ScriptResponseFiles`.

Reply:
Hum, doesn't appear to be any. The change is simple enough that I would not push for manual testing. Also the fact that this bug did not surface until now...